### PR TITLE
Tests: establish test/util.py and test/test_nmt.py

### PR DESCRIPTION
**Pull request by erlend-aasland**
Give NMT tests their own file, and establish a util.py for the test
suite for common stuff like the location of sample EDS files.

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Yak-shave in preparation for #500

**Comment by erlend-aasland** [comment]
Note that the NMT slave test now uses `setUp`/`tearDown` iso. their class equivalents, in order to provide a saner environment for the test cases.

**Comment by erlend-aasland** [comment]
Moreover, I removed some dead code.

File: test/util.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import os
+
+
+DATATYPES_EDS = os.path.join(os.path.dirname(__file__), "datatypes.eds")
+SAMPLE_EDS = os.path.join(os.path.dirname(__file__), "sample.eds")
+EDS_PATH = SAMPLE_EDS
```
Comment (acolomb) [inline test/util.py:6]: Shouldn't we get rid of this old name everywhere?  Might just as well adjust the other usages to the new name.

**Review by acolomb — APPROVED** [review]

File: test/util.py (hunk 1 above)
Comment (erlend-aasland) [reply test/util.py:6]: Sure; I just wanted to keep the diff to a minimum. I'll update the PR to consistently use the new name only.

Comment (erlend-aasland) [reply test/util.py]: Addressed with 0905d017c69ee9f3727dc3480591ddb073fc746b.

**Review by acolomb — APPROVED** [review]

**Comment by erlend-aasland** [comment]
Thanks for the review!

**Comment by acolomb** [comment]
Thanks for your continued contributions!

**Pull request merged**